features/org.wso2.carbon.identity.application.authenticator.oidc.server.feature/pom.xml

```diff
@@ -22,7 +22,7 @@
         <groupId>org.wso2.carbon.identity.outbound.auth.oidc</groupId>
         <artifactId>identity-application-auth-oidc</artifactId>
         <relativePath>../../pom.xml</relativePath>
-        <version>5.6.5-SNAPSHOT</version>
+        <version>5.6.6-SNAPSHOT</version>
```

Comment:
why these pom file versions are changed manually? please look into other pom files and remove those changes